src/backend/parser/gram.y

```diff
@@ -7139,6 +7139,74 @@ CreateMatViewStmt:
 					ctas->into->distributedBy = $13;
 					$$ = (Node *) ctas;
 				}
+/*****************************************************************************
+ *
+ *		QUERY :
+ *				CREATE DYNAMIC TABLE relname AS SelectStmt
+ *
+ *****************************************************************************/
+		| CREATE OptNoLog DYNAMIC TABLE create_mv_target SCHEDULE task_schedule AS SelectStmt opt_with_data OptDistributedBy
+				{
+					CreateTableAsStmt *ctas = makeNode(CreateTableAsStmt);
+					ctas->query = $9;
+					ctas->into = $5;
+					ctas->objtype = OBJECT_MATVIEW;
+					ctas->is_select_into = false;
+					ctas->if_not_exists = false;
+					/* cram additional flags into the IntoClause */
+					$5->rel->relpersistence = $2;
+					$5->skipData = !($10);
+					$5->dynamicTbl = true;
+					$5->schedule = $7;
+					ctas->into->distributedBy = $11;
+					$$ = (Node *) ctas;
+				}
+		| CREATE OptNoLog DYNAMIC TABLE create_mv_target AS SelectStmt opt_with_data OptDistributedBy
```

Comment:
without SCHEDULE is allowed ?

Reply:
Yes, please see https://github.com/apache/cloudberry/discussions/706#discussioncomment-11316615